nautobot_chatops/models.py

```diff
@@ -28,9 +27,16 @@ class CommandLog(BaseModel):
 
     command = models.CharField(max_length=64, help_text="Command issued")
     subcommand = models.CharField(max_length=64, help_text="Sub-command issued")
-    params = ArrayField(
-        ArrayField(models.CharField(default="", max_length=255)), default=list, help_text="user_input_parameters"
-    )
+
+    @staticmethod
+    def params_default():
+        """Provide default for the params field.
+
+        Per Django docs, the default must be a value or a callable.
+        """
+        return []
+
+    params = models.JSONField(default=params_default, help_text="user_input_parameters")
```

Comment:
Does `default=list` work without needing a custom `params_default` method?

Reply:
Maybe, I could be misreading:

The default can’t be a mutable object (model instance, list, set, etc.), as a reference to the same instance of that object would be used as the default value in all new model instances. Instead, wrap the desired default in a callable. For example, if you want to specify a default dict for JSONField, use a function:

Reply:
Right - you wouldn't want `default=[]` because `[]` is an instance of a list and it would be shared across all records. `default=list` should be fine because `list()` is a callable that returns a new instance of an empty list each time it is called.

Reply:
Great news! I will make the changes!